run_unittests.py

```diff
@@ -735,9 +735,7 @@ def assertRebuiltTarget(self, target):
             self.assertIn('Linking target {}'.format(target), ret)
         elif self.backend is Backend.vs:
             # Ensure that this target was rebuilt
-            clre = re.compile('ClCompile:\n [^\n]*cl[^\n]*' + target, flags=re.IGNORECASE)
             linkre = re.compile('Link:\n [^\n]*link[^\n]*' + target, flags=re.IGNORECASE)
-            self.assertRegex(ret, clre)
```

Comment:
Why was this removed?

Reply:
Because it's wrong.  See the commentary on commit d29cba1